seesaw/script/run_pipeline.py

```diff
@@ -161,7 +161,16 @@ def check_and_update():
     nonlocal_dict['timer'] = timer
 
     timer.start()
-
+def get_available_port(start_port=8001, end_port=8010):
```

Comment:
There should be two blank lines between functions to match the existing convention in the file